app/models/match.rb

```diff
@@ -0,0 +1,11 @@
+class Match < ApplicationRecord
+
+  belongs_to :season
+  belongs_to :winner, foreign_key: :winning_team_id
+  belongs_to :loser,  foreign_key: :losing_team_id
+
+  has_many   :games
+  has_one    :away_team
+  has_one    :home_team
+
```

Comment:
Consider adding:
* Banned Maps and who banned them - stats of the storm doesn't record this (it's external data)

Reply:
sorry additional comment:
This needs a date field for calendar creation, etc.

also: if we want to automate match announcements we'll need caster info i think elevated up to here (maybe the individual game isn't the right place). You'll want a primary caster (channel url) and then any co-casters/observers/production marked separately? Open for debate, I'm not a (very good) DB person so just one more thing to consider.

Reply:
Bringing this up so it's front of mind and not lost, from @ebshimizu :
`This needs a date field for calendar creation (when the match is scheduled).`

We might also be able to pre-determine when the matches are if we come up with a cool way to "load" a schedule - more on these crazy ideas later. 😎